### PR TITLE
Fix checkpoints.md

### PR DESCRIPTION
**Pull request by FKgk**
location: docs/site/en/r1/guide/checkpoints.md

remove img tag

this repo don't have first_train_calls.png and subsequent_calls.png

+++
Fix ko/guide/distributed_training.ipynb
change from tensorflow url to right url

+++
FIx en/guide/checkpoint.ipynb
change from tensorflow url to right url

### CODE REVIEW

**Comment by FKgk** [comment]
I signed it!

File: site/ko/guide/distributed_training.ipynb [hunk 1]
```diff
@@ -71,7 +71,7 @@
       "source": [
         "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/guide/distributed_training\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />TensorFlow.org에서 보기</a>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/distribute/keras\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />TensorFlow.org에서 보기</a>\n",
```
Comment (lamberta) [inline site/ko/guide/distributed_training.ipynb]: This is not correct. Please revert.

Also, please submit translation changes as a separate pull request so the community can review.

File: site/en/guide/checkpoint.ipynb [hunk 2]
```diff
@@ -53,7 +53,7 @@
       "source": [
         "\u003ctable class=\"tfo-notebook-buttons\" align=\"left\"\u003e\n",
         "  \u003ctd\u003e\n",
-        "    \u003ca target=\"_blank\" href=\"https://www.tensorflow.org/guide/checkpoint\"\u003e\u003cimg src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" /\u003eView on TensorFlow.org\u003c/a\u003e\n",
+        "    \u003ca target=\"_blank\" href=\"https://www.tensorflow.org/guide/checkpoints\"\u003e\u003cimg src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" /\u003eView on TensorFlow.org\u003c/a\u003e\n",
```
Comment (lamberta) [inline site/en/guide/checkpoint.ipynb]: Please revert this change. We're in the middle of switching the site around for TF2, the source of truth for these files is GitHub: https://github.com/tensorflow/docs/tree/master/site/en
And the site will follow shortly. Thanks

File: site/en/r1/guide/checkpoints.md [hunk 3]
```diff
@@ -63,9 +63,6 @@ classifier.train(
 As suggested by the following diagrams, the first call to `train`
 adds checkpoints and other files to the `model_dir` directory:
 
-<div style="width:80%; margin:auto; margin-bottom:10px; margin-top:20px;">
-<img style="width:100%" src="../images/first_train_calls.png">
```
Comment (lamberta) [inline site/en/r1/guide/checkpoints.md]: Let's not delete the images here. This can be changed to https://www.tensorflow.org/images/first_train_calls.png

File: site/en/r1/guide/checkpoints.md [hunk 4]
```diff
@@ -168,9 +165,6 @@ In other words, as the following illustration suggests, once checkpoints
 exist, TensorFlow rebuilds the model each time you call `train()`,
 `evaluate()`, or `predict()`.
 
-<div style="width:80%; margin:auto; margin-bottom:10px; margin-top:20px;">
-<img style="width:100%" src="../images/subsequent_calls.png">
```
Comment (lamberta) [inline site/en/r1/guide/checkpoints.md]: And this to https://www.tensorflow.org/images/subsequent_calls.png

File: site/en/r1/guide/checkpoints.md (hunk 3 above)
Comment (FKgk) [reply site/en/r1/guide/checkpoints.md]: Reverse the image tag and apply it.
Thanks

File: site/en/r1/guide/checkpoints.md (hunk 4 above)
Comment (FKgk) [reply site/en/r1/guide/checkpoints.md]: Reverse the image tag and apply it.
Thanks

File: site/en/guide/checkpoint.ipynb (hunk 2 above)
Comment (FKgk) [reply site/en/guide/checkpoint.ipynb]: I revert this change 
Thanks for information

File: site/ko/guide/distributed_training.ipynb (hunk 1 above)
Comment (FKgk) [reply site/ko/guide/distributed_training.ipynb]: I change correct

I don't know last sentence how can i do, can you explain it more easily?

Comment (FKgk) [reply site/ko/guide/distributed_training.ipynb]: I see the "The en/ directory is the source-of-truth." in "site/README.md"
I'm sorry I didn't check the document properly.

**Comment by lamberta** [comment]
Duplicate of https://github.com/tensorflow/docs/pull/1047
